Test/Tests/Component/NodeManager.Tests.ps1

```diff
@@ -61,7 +61,7 @@ function Test-NodeMgrConnectionWithController {
         hostname
     }
     $Out = Invoke-RestMethod ("http://$($ControllerIP):8089/Snh_ShowCollectorServerReq?")
-    $OurNode = $Out.ShowCollectorServerResp.generators.list.GeneratorSummaryInfo.source | Where-Object '#text' -Like $TestbedHostname
+    $OurNode = $Out.ShowCollectorServerResp.generators.list.GeneratorSummaryInfo.source | Where-Object '#text' -Like "$TestbedHostname*"
```

Comment:
Is it needed because domain name is added to hostname?

Reply:
Yes, but only when domain is set in instances.yaml, while deploying with CAD.